### ISSUE TITLE
git push dokku master gives fatal: 'chatanon.net' does not appear to be a repository

### ISSUE THREAD

**Issue by ROODAY** [progrium/dokku#965]
Here is what my console says:

```
git push dokku master
dokku@chatanon.net's password:
fatal: 'chatanon.net' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

I followed [this](https://gist.github.com/ngoldman/7287753) guide to set up dokku with digital ocean, and everything seems to be set up correctly. Hostname is correct, same as VHOST, I'm able to ssh both into root and dokku with no issue, and chatanon.net seems to be connected to my droplet (although it says that the server is down). But whenever I try to push, I get this error. I tried to look into other issues to see if there was an answer there, but nothing has seemed to work so far.

**Comment by michaelshobbs** [comment]
What do you get if you `ssh dokku@chatanon.net`? You'll need to use the ssh key you used in step 5 of that guide.

**Comment by ROODAY** [comment]
if I do straight up `ssh dokku@chatanon.net` it asks me for the dokku account's password on the server, which I set earlier and I'm able to get in. If I do `ssh -i R:\digitalocean_rsa.pub dokku@chatanon.net` then it makes me type the ssh keys passphrase 3 times, then still asks for dokku's password. Is this the issue?

**Comment by michaelshobbs** [comment]
I can't convince myself that it is the root cause. However, we can verify this by turning on tracing with `dokku trace on` and attempting to push again. A gist of the git push with tracing on should be quite helpful.

You can also configure your ssh client (and subsequently git) to use that pubkey. 

```
Host chatanon.net
  User dokku
  IdentityFile R:\digitalocean_rsa.pub
```

**Comment by ROODAY** [comment]
Ok I have no idea what happened but it started working. I'm happy that it is so I won't question it, but if anyone else has the same problem, what I did before I tried pushing again was that I was getting annoyed of having to keep typing my ssh passphrase for cmder, so I used Shooza's answer [here](https://github.com/bliker/cmder/issues/193) to fix that. After that, I did do the dokku trace on as @michaelshobbs requested, and I got a wall of text, but after it was done it did push and it worked. Not sure if the digitalocean server just needed some time or if it was a quirk with cmder's ssh.
